src/app/data-planes/sources.ts

```diff
@@ -42,8 +43,18 @@ const includes = <T extends readonly string[]>(arr: T, item: string): item is T[
   return arr.includes(item as T[number])
 }
 
-export const sources = (api: KumaApi, can: Can) => {
+export const sources = (source: Source, api: KumaApi, can: Can) => {
   return defineSources({
+    '/dataplanes': (params) => {
```

Comment:
At least for the moment can we change this to `/dataplanes/poll` so we know at a glance in application-land that its a polling one? Basically something so we know its not a basic "get me all the dataplanes once" one but is also polling.

Reply:
Done.

Reply:
Thanks!

Just while we are here, as little further note/potential follow up at some point.

I would really like to change all our sources to use `/dataplanes` (plural) for listings and actions on listings (i.e. `/dataplanes/poll`, `/dataplanes/by/type/` etc etc) and `/dataplane/:name` (singular) for items so we don't get clashes (generally you need to perform actions on listings but not on items)